docs/cloud-provider-specifics.md

```diff
@@ -181,75 +185,108 @@ Now that you have your Google Cloud credentials stored in a Secret, you need to
 
 ### Azure
 
+#### Kubernetes cluster prerequisites
+
+Ensure that the VMs for your agent pool allow Managed Disks. If I/O performance is critical,
+consider using Premium Managed Disks, as these are SSD backed.
+
 #### Service principal creation
 To integrate Heptio Ark with Azure, you should follow the instructions below to create an Ark-specific [service principal][17].
 
 1. If you do not have the `az` Azure CLI 2.0 locally installed, follow the [user guide][18] to set it up. Once done, run:
```

Comment:
The correct install guide for Azure CLI 2.0 is located at: https://docs.microsoft.com/en-us/cli/azure/install-azure-cli.

I would modify this line as follows:

> follow the [install guide](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) to set it up.